tox.ini

```diff
@@ -6,7 +6,8 @@ envlist = black,
           docstrings,
           py{36,37,38,39,launch},
           func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,docs,
-                  imports1,imports2,imports3,imports4,imports5,imports6,noml,grpc}-py37,
+                  imports1,imports2,imports3,imports4,imports5,imports6,
+                  imports7,imports8,imports9,imports10,imports11,noml,grpc}-py37,
```

Comment:
```suggestion
                  imports{1,2,3,4,5,6,7,8,9,10,11},noml,grpc}-py37,
```

Reply:
I think tox supports nested substitutions, let's give it a spin and see :)

Reply:
I think tox supports nested substitutions, let's give it a spin and see :)

Reply:
Looks like it does!